### PR TITLE
[EN-116] Joints

### PR DESCRIPTION
**Pull request by stepan-beresnev**

### CODE REVIEW

File: lib/joint/DistanceJoint.js [hunk 1]
```diff
@@ -142,14 +142,28 @@ function DistanceJoint(def, bodyA, bodyB, anchorA, anchorB) {
...
 DistanceJoint.prototype.getLocalAnchorA = function() {
   return this.m_localAnchorA;
-}
+};
+
+/**
+ * Sets the local anchor point relative to bodyA's origin.
+ */
+DistanceJoint.prototype.setLocalAnchorA = function(anchor) {
+  return this.m_localAnchorA = anchor;
```
Comment (yard) [inline lib/joint/DistanceJoint.js:151]: Is assignment, not copying safe here?

**Review by yard — CHANGES_REQUESTED** [review]
Many many setters simply assign vectors - not sure if that might be an issue?

File: lib/joint/DistanceJoint.js (hunk 1 above)
Comment (stepan-beresnev) [reply lib/joint/DistanceJoint.js:151]: getLocalAnchorA returns initial vector, not a clone, so in the same style did assignment here. Used it safely on engine level.

**Comment by stepan-beresnev** [comment]
Most of them are numbers, and setLocalAnchor*, which I used carefully.

**Review by yard — APPROVED** [review]

**Pull request merged**